FlatCache.xcodeproj/project.pbxproj

```diff
@@ -147,16 +147,17 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0900;
-				LastUpgradeCheck = 0900;
+				LastUpgradeCheck = 1000;
```

Comment:
Guess it will avoid Xcode 10 asking to upgrade the project again in future. But all I did is just click "OK"